.github/workflows/main.yml

```diff
@@ -30,6 +30,9 @@ jobs:
     services:
       consul:
         image: consul:1.9.3
+        env:
+          CONSUL_LOCAL_CONFIG: '{"acl": [{"default_policy": "allow", "enable_token_persistence": true, "enabled": true}], "services": [ {"address": "1.1.1.1", "checks": [], "id": "test-service-1", "name": "test-service", "port": 20001}, {"address": "2.2.2.2", "checks": [], "id": "test-service-2", "name": "test-service", "port": 20002}, {"address": "3.3.3.3", "checks": [], "id": "test-service-3", "name": "test-service", "port": 20003} ]}'
```

Comment:
It would be great to use the same testing setup as we do for local testing here. However, I anticipate that being a bit of work. Let's leave a comment/TODO?

Reply:
Agree. We would need to publish a Dockerfile to DockerHub and use a custom Docker image here. Will add a TODO.